File: src/main/java/ru/zdoher/japs/repositories/textbook/TextbookTypeRepository.java
package ru.zdoher.japs.repositories.textbook;

import org.springframework.data.mongodb.repository.ReactiveMongoRepository;
import org.springframework.stereotype.Repository;
import ru.zdoher.japs.domain.textbook.TextbookType;

@Repository
public interface TextbookTypeRepository
        extends ReactiveMongoRepository<TextbookType, String> {
}

File: src/main/java/ru/zdoher/japs/repositories/textbook/TextbookTypeRepository.kt
package ru.zdoher.japs.repositories.textbook

import org.springframework.data.mongodb.repository.ReactiveMongoRepository
import org.springframework.stereotype.Repository
import ru.zdoher.japs.domain.textbook.TextbookType

@Repository
interface TextbookTypeRepository : ReactiveMongoRepository<TextbookType, String>